Go 1.24 dropped the tlskyber `godebug` directive (and its corresponding code):
as it's being used to disable it here, we can retain current behaviour by
simply removing it.
---
diff --git a/go.mod b/go.mod
index ad50fc08..a6952efa 100644
--- a/go.mod
+++ b/go.mod
@@ -8,7 +8,7 @@ toolchain go1.23.5
 // This was causing errors with AWS Network Firewall
 // See https://github.com/DataDog/datadog-agent/issues/34323 for details.
 // This will be revisited once we update to 1.24.x
-godebug tlskyber=0
+// godebug tlskyber=0
 
 // v0.8.0 was tagged long ago, and appared on pkg.go.dev.  We do not want any tagged version
 // to appear there.  The trick to accomplish this is to make a new version (in this case v0.9.0)
